kore/src/Kore/ASTVerifier/SentenceVerifier.hs

```diff
@@ -502,4 +502,5 @@ parseAndVerifyAxiomAttributes indexModule attrs =
     parseAxiomAttributes' attrs >>= verifyAxiomAttributes indexModule
   where
     parseAxiomAttributes' =
-        Attribute.Parser.liftParser . Attribute.parseAxiomAttributes
+        Attribute.Parser.liftParser . Attribute.parseAxiomAttributes freeVariables'
```

Comment:
Here you need to use only the `FreeVariables` of the axiom that these attributes belong to. You will probably need to pass them in as another argument from the place where the sentences are verified.